nuxt.config.js

```diff
@@ -86,6 +92,9 @@ export default {
       },
     ],
   },
+  styleResources: {
+    scss: ['./assets/styles/*.scss'],
```

Comment:
Import the global.scss file here
```suggestion
    scss: ['./assets/styles/global.scss'],
```